.gitignore

```diff
@@ -17,6 +17,11 @@ project.properties
 xcuserdata/
 
 .gradle
+.vscode
+.settings
+
+*.classpath
+*.project
```

Comment:
This has nothing o do with HOLO card. If anything it needs to be in a separate PR with separate explanation